src/lib/fastDownward/fdFrontend.ts

```diff
@@ -264,3 +265,30 @@ export function findLearningPath<LU extends LearningUnit>({
 
 	return paths;
 }
+
+/**
+ *
```

Comment:
Add comments:
* For what is it for
* How does it work

Does this realy belong to fdFrontend (How to call FastDownward) ot should it be moved to PathPlaner (Frntend of the complete library)?

Reply:
I tried to follow the same way to call "search" function in fastDownward.ts. 
But that is correct, it shoud be in PathPlaner.
I moved it to PathPlaner (solved).